esrally/chart_generator.py

```diff
@@ -1033,7 +1033,7 @@ def segment_memory(title, environment, race_config):
                         "seperate_axis": 1,
                         "split_mode": "filters",
                         "stacked": "none",
-                        "filter": "environment:{} AND track:{}".format(environment, race_config.track),
+                        "filter": "environment:{} AND track:\"{}\"".format(environment, race_config.track),
```

Comment:
Maybe we should change the `.format()` invocations with f-strings in all places?